frontend/src/APIClients/CrudAPIClient.ts

```diff
@@ -0,0 +1,35 @@
+import baseAPIClient from "./BaseAPIClient";
+import USER_ACCESS_TOKEN_KEY from "../constants/AuthConstants";
+
+const create = async ({ formData }: { formData: any }): Promise<any> => {
+  const bearerToken = `Bearer ${localStorage.getItem(USER_ACCESS_TOKEN_KEY)}`;
+  try {
+    const { data } = await baseAPIClient.post("/entities", formData, {
+      headers: { Authorization: bearerToken },
+    });
+    return data;
+  } catch (error) {
+    return error;
+  }
+};
+
+const update = async ({ formData }: { formData: any }): Promise<any> => {
+  const bearerToken = `Bearer ${localStorage.getItem(USER_ACCESS_TOKEN_KEY)}`;
+  try {
+    const entityData = JSON.parse(JSON.stringify(formData));
+    delete entityData.id;
```

Comment:
Same comment as above about the function siganture, and this formData to object conversion should happen before this function is called. You can make the id a separate argument to this function so you can reuse the `EntityRequest` type